db/schema.rb

```diff
@@ -45,6 +45,17 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "mucked_homes", force: :cascade do |t|
+    t.string "name"
+    t.text "description"
+    t.string "phone"
+    t.string "pin"
+    t.string "lat"
```

Comment:
lat/lng should be floats if we're going to be able to show distance by geo

Also let's go with `latitude` and `longitude`. We can output them as `lat/lng` if the API demands

Reply:
I think the problem is that they are already Strings on the TexasRescueMap MongoDB. But yeah, I guess we can convert them as we import the data?